src/EditorFeatures/CSharpTest2/Recommendations/BoolKeywordRecommenderTests.cs

```diff
@@ -9,515 +11,515 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Recommendations
 {
     public class BoolKeywordRecommenderTests : KeywordRecommenderTests
     {
-        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
-        public async Task TestAtRoot_Interactive()
+        [WpfFact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
```

Comment:
Why  are the these WpfFact's now?  This is very much not good.  We did a bunch of a work to make sure recommender tests could run concurrently.  This undoes that.